ambari-funtest/pom.xml

```diff
@@ -527,16 +526,27 @@
       <groupId>org.apache.ambari</groupId>
       <artifactId>ambari-metrics-common</artifactId>
       <version>${project.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.httpcomponents</groupId>
```

Comment:
Are we sure this is not needed for Metrics?

Reply:
We directly depend on org.apache.httpcomponents:httpclient in ambari-funtest:
```
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ ambari-funtest ---
[INFO] org.apache.ambari:ambari-funtest:jar:2.6.1.0.0
[INFO] \- org.apache.httpcomponents:httpclient:jar:4.5.5:compile
```
The managed version (v4.5.5) will be available on the classpath so that Metrics can use its classes.